package.json

```diff
@@ -10,18 +10,14 @@
     "test": "test"
   },
   "dependencies": {
-    "amqplib": "0.5.1",
     "chpr-logger": "2.4.1",
-    "chpr-metrics": "1.1.0",
-    "co": "4.6.0",
-    "joi": "10.2.1",
-    "lodash": "4.14.2"
+    "lodash": "4.14.2",
+    "stakhanov": "0.2.1"
```

Comment:
It doesn't seem very safe to introduce a dependency with a beta version (even if it's our code)

Reply:
Bump to `0.3.0` after @jean-k-cp's fix in stakhanov

Reply:
we'll do this in a next version so we have both versions: one with stakhanov (no graceful shutdown) and one with the extra graceful shutdown, and we can bump to the one we think is safe 🙂 🛩